htdocs/install/mysql/tables/llx_fichinter.sql

```diff
@@ -39,6 +39,7 @@ create table llx_fichinter
   datee				date,						-- date de fin d'intervention
   datet				date,						-- date de terminaison de l'intervention
   duree				real,                       -- duree totale de l'intervention
+  signed            boolean  DEFAULT NULL,      -- signed status
```

Comment:
Can you use the type int default 0 ?
Rhevtype boolen is a pain when we need to introduce a third possible value...

Reply:
I had thought of using the following :

- default **NULL** : unknown/empty sign status for not showing anything for interventions pre feature
- **FALSE** : not signed
- **TRUE** : signed

I will update and propose to use int as following :

- default **-1** or **NULL** : unknown/empty sign status for not showing anything for interventions pre feature
- **0** : not signed
- **1** : signed (by client)

If needed futur values could be :

- **2** : signed by organisation/company
- **3** : signed by both parties